libkbfs/folder_branch_ops.go

```diff
@@ -4704,6 +4704,11 @@ func (fbo *folderBranchOps) applyMDUpdatesLocked(ctx context.Context,
 				return err
 			}
 		}
+		if rmd.IsRekeySet() {
```

Comment:
If the device sends an MD update and sets rekey=true and sets a paper prompt, I worry that its own update could unset the paper prompt that it just itself set.